docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx

```diff
@@ -30,3 +30,11 @@ Attempts to use unlicensed products will result in an error message and users wi
 ```code
 this Teleport cluster is not licensed for database access, please contact the cluster administrator
 ```
+
+# License Updates and Renewals
+
+If your license is updated for features and/or renewals an updated license file is required.  Download the updated license file
```

Comment:
Maybe we can copy the more detailed license file instructions from the Teleport Enterprise getting started guide (https://goteleport.com/docs/deploy-a-cluster/teleport-enterprise/getting-started/#obtain-your-license-file) into a partial and include these here?